src/components/singleBolgPage/index.css

```diff
@@ -0,0 +1,117 @@
+.articleHeader,
+.autherImage,
```

Comment:
typo in author.

typo in directory name

fix everywhere

Reply:
For changing the directory name I will change it with Routes when and make a new PR, because it might be confusing for you if I change everything in this PR